README.md

```diff
@@ -0,0 +1,35 @@
+# ArmCord Flatpak
+
+This is the flatpak for ArmCord. Due to sandbox limitations, it might not function as expected, so some ways to get it working are detailed below:
+
+## Discord Rich Presence
+### Native applications
+A solution that works short-term is to run `ln -sf $XDG_RUNTIME_DIR/{.flatpak/xyz.armcord.ArmCord/xdg-run,}/discord-ipc-0`. 
+For something longer lasting, run the following:
+
+```sh
+mkdir -p ~/.config/user-tmpfiles.d
+echo 'L %t/discord-ipc-0 - - - - .flatpak/xyz.armcord.ArmCord/xdg-run/discord-ipc-0' > ~/.config/user-tmpfiles.d/discord-rpc.conf
+systemctl --user enable --now systemd-tmpfiles-setup.service
+```
+Now, native applications will be able to use Rich Presence on every system start.
+
+### Flatpak applications
+TAKEN FROM https://github.com/flathub/com.discordapp.Discord/wiki/Rich-Precense-(discord-rpc)
```

Comment:
Just my opinion, there's no reason to let the user know that this is taken from something else. A comment in the README should be sufficient.

```suggestion
<!-- TAKEN FROM https://github.com/flathub/com.discordapp.Discord/wiki/Rich-Precense-(discord-rpc) -->
```